src/librustdoc/core.rs

```diff
@@ -424,18 +428,20 @@ pub fn run_core(options: RustdocOptions) -> (clean::Crate, RenderInfo, RenderOpt
                 // actually be loaded, just in case they're only referred to inside
                 // intra-doc-links
                 resolver.borrow_mut().access(|resolver| {
-                    for extern_name in &extern_names {
-                        resolver
-                            .resolve_str_path_error(
-                                DUMMY_SP,
-                                extern_name,
-                                TypeNS,
-                                LocalDefId { local_def_index: CRATE_DEF_INDEX }.to_def_id(),
-                            )
-                            .unwrap_or_else(|()| {
-                                panic!("Unable to resolve external crate {}", extern_name)
-                            });
-                    }
+                    sess.time("load extern crates", || {
```

Comment:
These should be `load_extern_crates`, we want to avoid spaces in these names for various reasons.

Reply:
Done. Note that the rustdoc passes have dashes and not underscores, is that fine?

Reply:
I would prefer to be consistent, but dashes don't seem bad. The main reason for avoiding spaces is so that e.g. URLs don't need escaping.